MetaMask/metamask-extension
# Fix #20648 - Show Linea ETH fiat value by default

## Explanation

* Fixes #20648

The selector that signals if we should show fiat conversions for a given network was not accounting for Linea, so the only way you'd see Linea fiat values is if you turned on the "Show fiat conversion on test networks" toggle in Advanced settings.  This PR shows fiat values for Linea at all times.

## Screenshots/Screencaps

<img width="720" alt="SCR-20230830-oywp" src="https://github.com/MetaMask/metamask-extension/assets/46655/0605b02e-4227-483e-922e-fccc81fc9e73">

## Manual Testing Steps

1.  Switch to Linea
2. See fiat value

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
